app-backend/db/src/main/resources/migrations/V9__Add_taskId_to_annotations.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+ALTER TABLE annotations
+ADD COLUMN task_id uuid;
```

Comment:
I think we could just do `ADD COLUMN task_id uuid REFERENCES public.tasks(id) ON DELETE SET NULL;` Also wondering if it complained about not using `public.annotations ` in the previous line.